### PR TITLE
format: clang tidy fix

### PR DESCRIPTION
**Pull request by hennna**
clang tidy fix

### CODE REVIEW

File: source/common/grpc/common.cc [hunk 1]
```diff
@@ -53,8 +53,8 @@ void Common::chargeStat(const Upstream::ClusterInfo& cluster, const std::string&
   chargeStat(cluster, "grpc", grpc_service, grpc_method, success);
 }
 
-Optional<Status::GrpcStatus> Common::getGrpcStatus(const Http::HeaderMap& trailers) {
```
Comment (htuch) [inline source/common/grpc/common.cc]: This is a weird one. Why doesn't clang-tidy like this? Semantically, the gRPC status comes from the trailers.

File: test/integration/integration.h [hunk 2]
```diff
@@ -171,7 +171,7 @@ class BaseIntegrationTest : Logger::Loggable<Logger::Id::testing> {
   void registerPort(const std::string& key, uint32_t port);
   uint32_t lookupPort(const std::string& key);
 
-  void sendRawHttpAndWaitForResponse(const char* http, std::string* response);
+  void sendRawHttpAndWaitForResponse(const char* http, std::string* raw_http);
```
Comment (htuch) [inline test/integration/integration.h]: Ditto: a rose is a rose?

File: source/common/grpc/common.cc (hunk 1 above)
Comment (hennna) [inline source/common/grpc/common.cc]: I think it's complaining that the .cc definition doesn't match the .h definition. Do you mean it was intentional in this case?

Comment (htuch) [inline source/common/grpc/common.cc]: Yep - the `grpc-status` header is provided in the trailers in gRPC http://www.grpc.io/docs/guides/wire.html

Comment (hennna) [inline source/common/grpc/common.cc]: Ok. I'll exclude from this PR.

**Pull request merged**